Gemfile

```diff
@@ -8,7 +8,7 @@ gem "winrm-fs"
 gem "activesupport", "~> 4.0"
 
 group :test do
-  gem "rake"
+  gem "rake", "< 12"
```

Comment:
I just tried updating to finstyle 1.5.0. It still isn't compatible with rake 12, and it picks up 11 more style offences :)